.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
If you could please remove this directory, I don't want to impose vscode on examples